common/lib/xmodule/xmodule/partitions/partitions.py

```diff
@@ -2,12 +2,23 @@
 
 from collections import namedtuple
 from stevedore.extension import ExtensionManager
+from xblock.fields import List
 
 # We use ``id`` in this file as the IDs of our Groups and UserPartitions,
 # which Pylint disapproves of.
 # pylint: disable=redefined-builtin
 
 
+class UserPartitionList(List):
+    """Special List class for listing UserPartitions"""
```

Comment:
_> I just hate to take up the space .. for things that are self-documenting_

You don't have to!  If it's self-documenting, you are smarter than the linter.  Just pragma (pylint-disable) the warning!  This is approved by Clean Code standards - so you'll still be a "conformant rebel".  :)